### ISSUE TITLE
Nanopolish index is not working

### ISSUE THREAD

**Issue by siupenyau** [jts/nanopolish#723]
Hi,

I am new in bioinformatics. I was trying to call variants after the whole genome sequencing of one bacteria sample versus its reference genome with nanopolish variants. When I tried to index the fast5 directory and the fastq reads:

nanopolish index -d /PATH-to-fast5/ /PATH-to-fastq/sample_merged.fastq

it did nothing without displaying any error message. Interestingly, when I randomly type "nanopolish abc", I also got the same response without any error message. Was that the installation of nanopolish in my case? The nanopolish version should be 0.11.1, installed in May 2019. My OS was Ubuntu 16.04 LTS 64 bit.

Thank you for your help in advance.

With best wishes,

Timothy Ng

**Comment by jts** [comment]
Hi,

It would be strange for nothing to be printed to the terminal, either some error messages, output or a "command not found" message should be printed. Can you confirm there are no messages at all (feel free to paste text from your terminal).

Jared

**Comment by siupenyau** [comment]
Thank you for your quick reply Jared. I checked a similar report somewhere. That was the problem of missing './'. After I had typed "./" before nanopolish, it worked fine.

I am now trying to call variants with the following command:
./nanopolish variants -m 0.4 -p 1 -t 4 -r /PATH-to-fastq/sample_merged.fastq -b /PATH-to-bam/sample_merged.bam -g reference.fasta -o /PATH-to-vcf/sample_merged.vcf

I saw an empty vcf was created in the target directory. May I confirm the variant calling was running in silent mode? I did not see any progress displaying on the screen.

Timothy Ng
